docs/user-guide/manage-environments.md

```diff
@@ -178,7 +178,25 @@ To inspect the deployed application in KubeRocketCI portal, take the following s
 
     ![Inspect application](../assets/user-guide/environments/application_terminal.png "Inspect application")
 
+### Delete Application
+
+There are two buttons in the environment which can be used in the portal:
+
+  ![Clean button](../assets/operator-guide/edp-portal-clean-application.png "Clean button")
+
+  * **Delete** - Deletes selected applications and associated resources.
+  * **Clean** - This button is useful when you need to perform custom cleanup actions, e.g. delete cloud resources, database, rollback transactions, etc.
+
+The **Delete** button works best when your application doesn't require deploying spicific resources, such as databases and external applications, or configuring the environment in a specific way.
```

Comment:
The **Delete** button is optimized for applications based on a single, simple helm chart that can be deployed independently without any specific dependencies.